### DISCUSSION TITLE
Is Discharge Disposition in EncounterEnd node recorded in csv output?

### DISCUSSION THREAD

**Discussion by samrere** [synthetichealth/synthea#1428] [Q&A] [answered]
Hi,
I can't seem to find the code I used for discharge disposition in the csv output? Thanks for any help!

**Comment by jawalonoski** [comment, marked as answer]
No, `encounters.csv` does not include discharge disposition.

https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary#encounters

That is why you can't find it.

Discharge disposition should get written to FHIR, if you use that exporter.

Not all the exporters are equivalent.
